### PR TITLE
Fixes for data races in logs

### PR DESCRIPTION
**Pull request by Mixaster995**
Signed-off-by: Mikhail Avramenko <avramenkomihail15@gmail.com>

## Description

Refactored usage of log fields to prevent data races

## Issue link

fixes https://github.com/networkservicemesh/cmd-forwarder-vpp/pull/372

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [x] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [x] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

File: pkg/networkservice/chains/nsmgr/scale_test.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ import (
 func TestCreateEndpointDuringRequest(t *testing.T) {
 	t.Cleanup(func() { goleak.VerifyNone(t) })
 
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5*10)
```
Comment (edwarnicke) [inline pkg/networkservice/chains/nsmgr/scale_test.go:41]: Why are we suddenly 10x-ing all of the timeouts? What are we doing that is bloating the latency by 10x?

Comment (Mixaster995) [reply pkg/networkservice/chains/nsmgr/scale_test.go:41]: It seems, that enabling full logs in tests running with race flag, increases time for tests. That's not something i'm going to actually push to sdk code, some changes only have been made for testing, so i'm converted this PR to draft, until it's ready.

**Review by denis-tingaikin — COMMENTED** [review]
LGTM. Could you remove all debug logic?

**Comment by Mixaster995** [comment]
@denis-tingaikin removed all unrelated changes from PR

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**